torchrl/modules/tensordict_module/exploration.py

```diff
@@ -481,18 +483,19 @@ def step(self, frames: int = 1) -> None:
     def forward(self, tensordict: TensorDictBase) -> TensorDictBase:
         tensordict = super().forward(tensordict)
         if exploration_type() == ExplorationType.RANDOM or exploration_type() is None:
-            if "is_init" not in tensordict.keys():
+            if self.is_init_key not in tensordict.keys(True, True):
```

Comment:
I would like for the possibility of the init_key being nested. Is there a way we can run this check just once and then skip?

Reply:
I solved this